frontend/public/components/machine-set.tsx

```diff
@@ -299,11 +363,39 @@ export const MachineSetList: React.SFC = (props) => {
               obj.metadata.namespace,
             )}/machines`}
           >
-            {getReadyReplicas(obj)} {t('public~of')} {getDesiredReplicas(obj)}{' '}
-            {t('public~machines')}
+            {t('public~{{numReplicas}} of {{numDesiredReplicas}} machine', {
+              numReplicas: getReadyReplicas(obj),
+              numDesiredReplicas: getDesiredReplicas(obj),
+              count: getDesiredReplicas(obj),
+            })}
           </Link>
         </TableData>
         <TableData className={tableColumnClasses[3]}>
+          {getMachineSetInstanceType(obj) || '-'}
+        </TableData>
+        <TableData className={tableColumnClasses[4]}>
+          {relatedNodes.reduce(
+            (cpuSize, node) => cpuSize + convertToBaseValue(node.status?.capacity.cpu),
+            0,
+          )}
+        </TableData>
+        <TableData className={tableColumnClasses[5]}>
+          {t('public~{{memory}} GiB', {
+            memory: Math.round(
+              parseInt(
+                formatBytesAsGiB(
+                  relatedNodes.reduce(
+                    (memorySize, node) =>
+                      memorySize + convertToBaseValue(node.status.capacity.memory),
```

Comment:
We should be consistent in how we use optional chaining. You use it for `node.status` above, but not here.